packages/teams-js/src/public/pages.ts

```diff
@@ -29,6 +29,19 @@ export namespace pages {
     sendMessageToParent('returnFocus', [navigateForward]);
   }
 
+  /**
+   * @hidden
+   * Registers a handler when focus needs to be passed from teams to the place of choice on app.
+   *
+   * @param handler - The handler to invoked by the app when they want the focus to be in the place of their choice.
+   *
+   * @internal
+   */
+  export function registerFocusEnterHandler(handler: (navigateForward: boolean) => void): void {
```

Comment:
The other register*Handler functions seem to go in pages.config, would we want this in there as well?

Reply:
Discussed offline. This one doesn't have to do with config while those registerHandler functions in pages.config do! Simple, no need to move.

Reply:
Closing this comment per our convo. FocusEnter should stays in top-level in pages. :)